browser/src/CopyNumberVariantPage/CNVPopulationsTable.tsx

```diff
@@ -80,22 +81,24 @@ type OwnPopulationsTableProps = {
     }[]
   }[]
   initiallyExpandRows?: boolean
+  variant: CopyNumberVariant
 }
 
 type CNVPopulationsTableState = any
 
 type CNVPopulationsTableProps = OwnPopulationsTableProps & typeof CNVPopulationsTable.defaultProps
 
 export class CNVPopulationsTable extends Component<
-  CNVPopulationsTableProps,
+  CNVPopulationsTableProps & { variant: CopyNumberVariant},
```

Comment:
It should be simple enough to fold all these various `WhateverProps` together to make things a little clearer, but let's save that for later. This whole business with `OwnWhateverProps & typeof Whatever.defaultProps` that you see here and there in the code is just an artifact of the automatic conversion of the codebase from JS to TS a long time ago, no need to look for any deep meaning in it. I clean these up when I am working near one but it's not a priority.

Reply:
Thanks for pointing this out, when I go back and go through the actual code review of the CNVs I'll make sure to come back to this.